### PR TITLE
services: make reflection test able to re-root proto

### PR DESCRIPTION
**Pull request by carl-mastrangelo**

### CODE REVIEW

File: services/src/test/java/io/grpc/protobuf/service/ProtoReflectionServiceTest.java [hunk 1]
```diff
@@ -214,7 +217,10 @@ public void fileByFilenameForMutableServices() throws Exception {
     ServerReflectionRequest request =
         ServerReflectionRequest.newBuilder()
             .setHost(TEST_HOST)
-            .setFileByFilename("io/grpc/reflection/testing/dynamic_reflection_test_depth_two.proto")
+            .setFileByFilename(
+                FILE_NAME_PREFIX
+                    + "io/grpc/reflection/testing/"
+                    + "dynamic_reflection_test_depth_two.proto")
```
Comment (ericgribkoff) [inline services/src/test/java/io/grpc/protobuf/service/ProtoReflectionServiceTest.java]: Does this string need to be split over two lines?

Comment (carl-mastrangelo) [inline services/src/test/java/io/grpc/protobuf/service/ProtoReflectionServiceTest.java]: done.

**Comment by carl-mastrangelo** [comment]
I ran [google-java-format](https://github.com/google/google-java-format) on it.

**Comment by ejona86** [comment]
> I ran google-java-format on it.

How about no? File-wide reformatting is not okay.

**Pull request closed without merging**